untitled.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Why is this untitled? What are all these yaml files for?

Reply:
Those files were for GUI for planning out pathways - those are old so we are just going to delete those.